pkg/kontext/dockerignore.go

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
I'd split off a separate ignore package so the public/private API is clearer.  This also means we can use `pkg/ignore/testdata` which is slightly more idiomatic of Go.

Reply:
fixed in the latest push